### PR TITLE
adding subtract_ to torch frontend

### LINKED ISSUE #13992: subtract_

### PR DESCRIPTION
**Pull request by vtsamit**
Close #13992

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 1]
```diff
@@ -3207,8 +3243,6 @@ def test_torch_instance_arccos_(
         method_flags=method_flags,
         frontend=frontend,
         on_device=on_device,
-    )
```
Comment (binh234) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:3210]: Hi @vtsamit @soma2000-lang, please revert the changes here, it seems like you accidentally deleted the close bracket.

Comment (vtsamit) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:3210]: Really sorry sir. Can this PR be opened once again so that I can add the close bracket?
